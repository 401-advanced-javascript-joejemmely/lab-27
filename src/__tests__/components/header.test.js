import React from 'react';
import Header from '../../components/header/header';

describe('Header', () => {
  it('renders', () => {
    const mountedSimple = shallow(<Header />);
    expect(mountedSimple.find('header')).toBeTruthy();
  });
});
