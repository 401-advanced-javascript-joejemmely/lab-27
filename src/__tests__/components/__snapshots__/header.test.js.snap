// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header rendering follows the snapshot 1`] = `
<header>
  <h1
    className="css-1c7hblg"
  >
    Counters R Fun
  </h1>
</header>
`;
